### PR TITLE
[docs-only] Add a few words about idm and how to enable it

### PR DESCRIPTION
**Pull request by rhafer**
## Description

More an more people are looking into libregraph/idm related things. This adds some words about how to use idm with the current code (instead of spreading the info across multiple github issues)

### CODE REVIEW

File: docs/extensions/idm/setup.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+geekdocCollapseSection: true
+---
+
+{{< toc >}}
+
+## Using ocis with libregraph/idm
+
+Currently ocis will still run the accounts and glauth services to manage users. Until the default is switched
+to libregraph/idm, ocis has to be started with a custom configuration in order to use libregraph/idm as
+the users and groups backend (this setup also disables the glauth and accounts serice:
```
Comment (phil-davis) [inline docs/extensions/idm/setup.md:17]:
```suggestion
the users and groups backend (this setup also disables the glauth and accounts service):
```

Comment (rhafer) [reply docs/extensions/idm/setup.md]: fixed

File: docs/extensions/idm/setup.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+geekdocEditPath: edit/master/docs/extensions/idm
+geekdocFilePath: setup.md
+geekdocCollapseSection: true
+---
+
+{{< toc >}}
+
+## Using ocis with libregraph/idm
+
+Currently ocis will still run the accounts and glauth services to manage users. Until the default is switched
```
Comment (pascalwengerter) [inline docs/extensions/idm/setup.md:15]:
```suggestion
Currently, oCIS still runs the accounts and glauth services to manage users. Until the default is switched
```

File: docs/extensions/idm/setup.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+to libregraph/idm, ocis has to be started with a custom configuration in order to use libregraph/idm as
+the users and groups backend (this setup also disables the glauth and accounts service:
```
Comment (pascalwengerter) [inline docs/extensions/idm/setup.md:17]:
```suggestion
the users and groups backend (this setup also disables the glauth and accounts service):
```

**Review by pascalwengerter — COMMENTED** [review]
Spotted some tiny annoyances 🕵🏽

**Review by phil-davis — APPROVED** [review]

**Pull request merged**